src/tag/tag-filter.component.ts

```diff
@@ -2,37 +2,59 @@ import {
 	Component,
 	Output,
 	EventEmitter,
-	HostBinding
+	HostBinding,
+	Input,
+	ViewChild,
+	ElementRef
 } from "@angular/core";
 import { Tag } from "./tag.component";
 
 @Component({
 	selector: "ibm-tag-filter",
 	template: `
-		<ng-content></ng-content>
-		<svg
-			(click)="close.emit()"
-			focusable="false"
-			preserveAspectRatio="xMidYMid meet"
+		<span #label class="bx--tag__label" [title]="label.innerHTML">
+			<ng-content></ng-content>
+		</span>
+		<button
 			class="bx--tag__close-icon"
-			style="will-change: transform;"
-			xmlns="http://www.w3.org/2000/svg"
-			aria-label="Clear filter"
-			width="16"
-			height="16"
-			viewBox="0 0 16 16"
-			role="img">
-			<path d="M12 4.7l-.7-.7L8 7.3 4.7 4l-.7.7L7.3 8 4 11.3l.7.7L8 8.7l3.3 3.3.7-.7L8.7 8z"></path>
-		</svg>
+			(click)="close.emit()"
+			[disabled]="disabled"
+			[attr.aria-labelledby]="id || tagId"
+			[title]="title">
+			<span class="bx--visually-hidden">{{title}}</span>
+			<svg ibmIconClose size="16"></svg>
+		</button>
 	`
 })
 export class TagFilter extends Tag {
+	static tagFilterCount = 0;
+	@Input() tagId = `tag-filter-${TagFilter.tagFilterCount}`;
+	@Input() title = "Clear Filter";
```

Comment:
Is this the title for the whole tag or for the tag close button?

Reply:
Tag close button... I'll make it more descriptive.

Reply:
> Thanks for taking this on! It's definitely much needed ... Just a few comments/questions 😄

I appreciate the feedback! I was _trying_, to some extent, to keep in line with a similar approach they're using in the React library, BUT I think I'll take a stab at updating this to be more in line w/what I'd want as a developer and that'll probably address most, if not all, of the concerns raised here. 

Thanks for the review!! 🎉